### PR TITLE
Add info re: current_user scope rate limits

### PR DESCRIPTION
**Pull request by khorne3**
https://docs-content-staging-pr-8397.herokuapp.com/docs/api/management/v2/get-access-tokens-for-spas

https://docs-content-staging-pr-8397.herokuapp.com/docs/policies/rate-limits

### CODE REVIEW

File: articles/api/management/v2/get-access-tokens-for-spas.md [hunk 1]
```diff
@@ -38,6 +38,16 @@ With a Management API Token issued for a SPA, you can access the following scope
 | `create:current_user_device_credentials` | [POST /api/v2/device-credentials](/api/management/v2#!/Device_Credentials/post_device_credentials) |
 | `delete:current_user_device_credentials` | [DELETE /api/v2/device-credentials/{id}](/api/management/v2#!/Device_Credentials/delete_device_credentials_by_id) |
 
+### Rate Limits
+
+Please note that the above scopes (and therefore limits) are rate limited. By default, the rate limit is **disabled** (that is, the controlling tenant-level flag `dedicated_rate_limit_for_current_user_scope` is set to `false`). When `dedicated_rate_limit_for_current_user_scope = false`, Auth0 makes a log entry whenever the rate limit is exceeded:
```
Comment (siacomuzzi) [inline articles/api/management/v2/get-access-tokens-for-spas.md]: flag and logs are internal only

Comment (panga) [reply articles/api/management/v2/get-access-tokens-for-spas.md]: +1 please delete any flag references, the default value will be true

**Comment by michaelwallen** [comment]
@khorne3 Thanks you this all looks great:) 👍

**Comment by siacomuzzi** [comment]
@michaelwallen why are we not saying that the new rate limit is "10 requests per minute per user"?
IMO it's the first thing a customer will ask, right?

**Comment by michaelwallen** [comment]
@siacomuzzi Good point. @khorne3 Your thoughts on adding this?

**Comment by khorne3** [comment]
@siacomuzzi and @michaelwallen In the past, we have avoided providing specific numbers, but I would be happy to include this information if you would like. What do you think?

**Comment by siacomuzzi** [comment]
I think we should provide the numbers (we already are doing it with the other limits: https://docs-content-staging-pr-8397.herokuapp.com/docs/policies/rate-limits)

**Comment by khorne3** [comment]
@siacomuzzi Added rate limit info. Thanks!

**Review by panga — APPROVED** [review]

**Pull request merged**